app/config.py

```diff
@@ -1,22 +1,11 @@
 import json
-import discord
-
 import os
 from dotenv import load_dotenv
-load_dotenv()
 
-from app.bot.commands_manager import CommandsManager
+load_dotenv()
 
 TOKEN = os.getenv("TOKEN")
 DEBUG = os.getenv("DEBUG", 'true') == 'true'
 
-class AttrDict:
```

Comment:
rest in piece you will be missed

Reply:
this was a pretty smart idea, actually, but i don't think that was very pythonic, and it confused linters _very strongly_, not even in terms of checking for code style guidelines, but in terms of checking if a specific attribute exists (it just screamed at every usage of AttrDict, because it couldn't verify anything)

using a simple dict fixed that issue, and didn't add, like, a big amount of code or something

Reply:
makes sense.